internal/parser/parser/docx_numbering.go

```
@@ -0,0 +1,659 @@
+		return parts
+	}
+	for _, file := range zr.File {
+		if !wanted[file.Name] {
+			continue
+		}
+		r, err := file.Open()
+		if err != nil {
+			continue
+		}
+		content, err := io.ReadAll(r)
+		r.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `r.Close` is not checked

(errcheck)